Architecture_Engine/Query/OutlineElements1D.cs

```diff
@@ -47,6 +47,12 @@ public static partial class Query
         [Output("outlineElements", "A collection of outline 1D elements")]
         public static List<IElement1D> OutlineElements1D(this Room room)
         {
+            if(room == null)
+            {
+                BH.Engine.Reflection.Compute.RecordError("Cannot query the outline 1D elements of a null room.");
+                return null;
```

Comment:
Why null and not an empty list?
```suggestion
                return new List<IElement1D>();
```